features/autoconsent/banner.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Autoconsent Cosmetic Test</title>
```

Comment:
This should be in line with how we call it on the index page so "Cookie consent popup test".